.github/workflows/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -x
+
+maker=$1
+device=$2
+
+mydir=$(dirname $0)
+source ${mydir}/setup_env.sh
+
+print "======================================== Build"
+make -j8 || exit 10
+
+print "======================================== Install"
+make -j8 install || exit 11
+ls -R ${top}/install
```

Comment:
Just to make sure, the var `top` exists elsewhere?

Reply:
Set in setup_env.sh